vscode-dotnet-runtime-library/src/Utils/IssueReporter.ts

```diff
@@ -0,0 +1,21 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+import { IIssueContext } from './IIssueContext';
+
+const issuesUrl = `https://github.com/dotnet/vscode-dotnet-runtime/issues`;
+
+export function formatIssueUrl(error: Error, context: IIssueContext): string {
+    context.logger.dispose(); // Ensure log file is up to date
+    const issueBody = `<!-- IMPORTANT: Please be sure to remove any private information before submitting and attach the log file located at ${ context.logger.getFileLocation() }. -->

```

Comment:
Can you provide an example of what this would look like for a common error? I worry that there wont be enough identifying information to understand where errors are coming from for all of the components that create new processes etc.

Reply:
Here's a screenshot of the issue message:

![Screenshot (21)](https://user-images.githubusercontent.com/16010855/74489299-c157fd80-4e79-11ea-92cd-c880f60047b2.png)

And when you paste the error message (in this case, we failed to resolve the version `foo`):

![Screenshot (20)](https://user-images.githubusercontent.com/16010855/74488479-35dd6d00-4e77-11ea-81f1-431b1fbf6382.png)

I've included a note about attaching the log file in the original message, which includes more debugging details. Do you think that's sufficient?

Reply:
Looks freaking awesome!

Reply:
Let me know when you want me to do another pass at the PR. I typically notify reviewers that I'm ready for another pass by commenting 🆙 📅 on my PR.

Reply:
It's ready for another pass now- thanks!